### PR TITLE
Javascript javascript3 week1

### PR DESCRIPTION
**Pull request by SvitlanaBurlaka**

### CODE REVIEW

**Comment by SvitlanaBurlaka** [comment]
Homework for Javascript javascript3 week1

File: javascript/javascript3/week1/giphy api/giphy.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            )
+            .then((response) => response.json())
+            .then((content) => {
+                divTag.innerHTML = "";
+                for (let i = 0; i < limitOfGifNumber; i++) {
+                    divTag.innerHTML += `<img src ="${content.data[i].images.fixed_width.url}"><br/>`;
+                }
+            });
+    }
+});
```
Comment (Falafelqueen) [inline javascript/javascript3/week1/giphy api/giphy.js:28]: Works great !

File: javascript/javascript3/week1/index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <input type="text" textholder="write city name">
+    <button class="submit">Weather</button>
+    <button id="currentPosition">Get current position</button>
```
Comment (Falafelqueen) [inline javascript/javascript3/week1/index.html:15]: Just a side note : The camelCase is preferred way of naming your variables and functions in JS, however the convention for html would be to name your html attributes either all in lower case or connected with a "-". 👩🏻‍🎨

```
<button id="current-position"></>
```

File: javascript/javascript3/week1/index.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    img.src = `
+     http://openweathermap.org/img/wn/${data.weather[0].icon}@2x.png
+     `;
+    document.querySelector(".mainContainer").appendChild(img);
+    // Wind speed
+    wind.innerHTML = `Wind speed ${data.wind.speed}`;
+    //How clowdy it is
+    description.innerHTML = data.weather[0].description;
+    // When sunrise and sunset is
+    // For sunrise
```
Comment (Falafelqueen) [inline javascript/javascript3/week1/index.js:50]: Great logic there !! Just a small tip to save some lines : maybe you could create a new function doing this so you don't have to do the same for both sunrise and sunset.🥷

```
// for example and then you only call the function when assigning the innerHTML to your sunrise and sunset
function fixTheTime(input){
  const dateObject = new Date(input * 1000)
  return dateObject.toLocaleString("de-DE", {
    hour: "numeric",
    minute: "numeric",
  });
}

```

File: javascript/javascript3/week1/index.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+        fetch(
+                `https://api.openweathermap.org/data/2.5/weather?lat=${latitudeTwo}&lon=${longitudeTwo}&appid=141f299896d9c32c851d3c2c2311a11e&units=metric`
+            )
+            .then((response) => response.json())
+            .then((data) => {
+                printWeather(data);
+                initMap(latitudeTwo, longitudeTwo);
+            });
+    });
+});
```
Comment (Falafelqueen) [inline javascript/javascript3/week1/index.js:111]: Awesome, works great ! 🦸‍♀️

**Review by Falafelqueen — APPROVED** [review]
Great job Svitlana and well done on the optionals as well ! I left minor comments, but no need to be fixing anything, those are more for your info and future reference 👩🏼‍💻🦸‍♀️

**Pull request merged**